### PR TITLE
Features/actor page

### PR DESCRIPTION
**Pull request by marcin10lw**
Check out actor page,
I also changed on hover box-shadow color in actor and movie tiles

### CODE REVIEW

File: src/features/people/ActorPage/About/styled.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    "i p";
+  padding: 40px;
+  column-gap: 40px;
+
+  @media (max-width: ${bpMobile}px) {
+    column-gap: normal;
+    gap: 16px;
+    padding: 16px;
+  }
+`;
```
Comment (mikolajgraczyk) [inline src/features/people/ActorPage/About/styled.js]: I think you forgot about white background accordingto figma 😉

File: src/features/people/ActorPage/About/styled.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  width: 100%;
+  border-radius: 5px;
+  grid-area: 1 / 1 / 4 / 2;
+
+  @media (max-width: ${bpMobile}px) {
+    grid-row: 1 / 2;
+  }
+`;
+
+export const ActorInfo = styled.div``;
```
Comment (mikolajgraczyk) [inline src/features/people/ActorPage/About/styled.js]: Do we need this? 😃

**Review by mikolajgraczyk — CHANGES_REQUESTED** [review]
Really good job. Our site is getting more and more advanced

**Review by HelkaPL — APPROVED** [review]

File: src/features/people/ActorPage/Crew/index.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { useSelector } from "react-redux";
+import { Container } from "../../../../common/Container";
+import { GridList } from "../../../../common/GridList";
+import TilesSection from "../../../../common/TilesSection";
+import MovieTile from "../../../movies/MovieTile";
+import { selectMoviesCrew } from "../actorSlice";
+
+const Crew = () => {
+  const moviesCrew = useSelector(selectMoviesCrew);
+  console.log(moviesCrew);
```
Comment (mikolajgraczyk) [inline src/features/people/ActorPage/Crew/index.js]: One more thing. I spotted console.log that is not deleted

**Review by mikolajgraczyk — APPROVED** [review]

**Review by HelkaPL — APPROVED** [review]
Don't forget to correctly resolve merge conflict :)

**Pull request merged**